core/pkg/utils/build_notifications.go

```diff
@@ -0,0 +1,46 @@
+package utils
+
+import (
+	"reflect"
+
+	"github.com/open-feature/flagd/core/pkg/model"
+)
+
+func BuildNotifications(oldFlags, newFlags map[string]model.Flag) map[string]interface{} {
```

Comment:
Does it make sense to 

1. Create a package called `notifications`
2. Define the type Notifications as `map[string]any`
3. Rename this function to `NewFromFlags` or `New` and returns the defined type

?